full-stack-asset-transfer-guide/.github/workflows/asset-tx-typescript-contract-image.yaml

```diff
@@ -0,0 +1,24 @@
+name: Asset Tx Contract Image CI
```

Comment:
I assume these github workflows would move up to root directory (could do that in a 2nd PR...).

Reply:
Yes.  Trying not to muddy the waters in this initial PR.

For CI this will be a good opportunity to set up a beach head for running some (not all) of the sample tests over at GHA.